src/osd/OSD.cc

```diff
@@ -8476,6 +8477,16 @@ void OSDService::_maybe_queue_recovery() {
 
 bool OSDService::_recover_now(uint64_t *available_pushes)
 {
+  if (ceph_clock_now() < defer_recovery_until) {
+    dout(15) << "_recover_now defer until " << defer_recovery_until << dendl;
```

Comment:
@tchaikov  I change others.But can not change this. Because if I change it to the __func__  one ci test can not passed. It is very surprise me that. So i keep it here as the same.  The ci can not pass is test_objectstore_memstore.sh.